src/settings-ui/Settings.UI/Views/MeasureToolPage.xaml

```diff
@@ -30,22 +35,28 @@
                         </controls:Setting.ActionContent>
                     </controls:Setting>
                 </controls:SettingsGroup>
-                
+
                 <controls:SettingsGroup x:Uid="MeasureTool_Settings" IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}">
-                   
+
                     <controls:Setting x:Uid="MeasureTool_ContinuousCapture" Icon="&#xE7FB;">
                         <controls:Setting.ActionContent>
                             <ToggleSwitch IsOn="{x:Bind ViewModel.ContinuousCapture, Mode=TwoWay}" x:Uid="MeasureTool_ContinuousCapture_ToggleSwitch" />
                         </controls:Setting.ActionContent>
                     </controls:Setting>
+                    <muxc:InfoBar
+                        Severity="Warning"
+                        x:Uid="MeasureTool_ContinuousCapture_Information"
+                        IsOpen="True"
+                        IsTabStop="True"
+                        IsClosable="False"
+                        Visibility="{Binding Mode=OneWay, Path=ContinuousCapture, Converter={StaticResource BoolToVisibilityConverter}}" />
```

Comment:
is it hiding if you disable the toy or do we need a secon isopen property binding against `isRulerEnabled`?

Reply:
We need both. Otherwise we have a ui glitch. We need one check for setting and one for disabling toy. we did the same on other warnings.

Reply:
I think that's only on PowerLauncher, from what I see. on other "muxc:InfoBar" instances.

Reply:
For shortcut overwiev @jay-o-way is working on a fix.

Reply:
@jaimecbernardo "disabled" is referring to a visual style, that seems to be a bug in the control istelf: https://github.com/microsoft/PowerToys/pull/19592#discussion_r950869730
A manual fix needs a bit of tinkering.

Reply:
@jaimecbernardo
Can you please fix it by changing the `isOpen` value to the property for the setting and updating the `visibility` property to property for the Toy enablement state.

Reply:
Fixed in PR #20184.